provider.tf

```diff
@@ -1,4 +1,4 @@
 provider "aws" {
-  region  = "eu-north-1"
+  region = "eu-north-1"
 }
  
```

Comment:
could you please, add a newline at the end of file.

Reply:
extra line added